pandas/tests/reshape/test_pivot.py

```diff
@@ -901,6 +901,86 @@ def _check_output(
             totals = table.loc[("All", ""), item]
             assert totals == self.data[item].mean()
 
+    @pytest.mark.parametrize(
+        "columns, aggfunc, values, expected_columns",
+        [
+            (
+                "A",
+                np.mean,
+                [[5.5, 5.5, 2.2, 2.2], [8.0, 8.0, 4.4, 4.4]],
+                Index(["bar", "All", "foo", "All"]),
+            ),
+            (
+                ["A", "B"],
+                np.sum,
+                [[9, 13, 22, 5, 6, 11], [14, 18, 32, 11, 11, 22]],
+                MultiIndex.from_tuples(
+                    [
+                        ("bar", "one"),
+                        ("bar", "two"),
+                        ("bar", "All"),
+                        ("foo", "one"),
+                        ("foo", "two"),
+                        ("foo", "All"),
+                    ],
+                    names=["A", "B"],
+                ),
+            ),
+            (
+                ["A", "B", "C"],
```

Comment:
Does this test anything different from the test preceding it? It not I think can be removed

Reply:
thanks for your quick response! @WillAyd 

emm, this shows different number of levels for `columns`, is this a bit redendunt? if so, i will remove

Reply:
I think can be removed; the one preceding already tests for a multi index

Reply:
ok, sure!